src/errors.js

```diff
@@ -9,10 +9,13 @@ export class NotImplementedError extends Error {
 
 export class HttpError extends Error {
   constructor(message, status) {
-    super(message);
-
-    this.message = message;
-    this.status = status;
-    this.name = 'HttpError';
+    super(message.message);
+    this.name = this.constructor.name;
+    if (typeof Error.captureStackTrace === 'function') {
+      Error.captureStackTrace(this, this.constructor);
+    } else {
+      this.stack = new Error(message).stack;
+    }
+    this.stack = new Error().stack;
```

Comment:
doesn't it override the assignment from two lines before? 🤔

Reply:
nice catch, it's removed